notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>Let's say I wanted to use GRURec, Caser or ASVD, I just have to import the correct class and the rest of the notebook will be the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes, exactly.

Reply:
this is really amazing Jianxun, honestly, really good job